Update README.md (#3)
reanimation47/2d-platformer-project: README.md

# 2D-PLATFORMER-PROJECT

a platform game, which is a genre of video games that involve guiding a character through a series of levels while jumping, running, and dodging obstacles. The player controls the character and must navigate through various challenges and puzzles to progress through the game.
## Feature

This game has 02 main modes:
- Levels
 
The levels will include players and obstacles. Players need to find a way to pass obstacles and monsters to reach the destination (may add a requirement to collect enough items or keys lock, or kill all monsters, .. depending on the game screen)
- Alive

Players need to survive as long as possible (or eat as many items as possible?) while the
Obstacles and monsters spawn more and more with increasing difficulty until the player dies
(the game ends)

> Furthermore,this game allow user customize the character. Each character will have different attributes (movement speed, ..) & special abilities (double jump, dash, ... 

Reviewer: ```suggestion
Furthermore,this game will allow user to select different characters. Each character will have different attributes (movement speed, ..) & special abilities (double jump, dash, ... 
```

## Technology

Framework/Game Engine: Unity (C#)

> Applying the [CI/CD](https://docs.gitlab.com/ee/ci/)  approach to development: Github Action and GameCI

Reviewer: ```suggestion
Applying the [CI/CD](https://game.ci) approach to development: Github Action and GameCI
```

The Unity development environment itself has enough capabilities to manage & encapsulate dependencies.

> ## Software Requirement
> 
> - Unity 2021.3.22f1
> - [Unity license](https://github.com/reanimation47/2d-platformer-project/actions/workflows/activation.yml) for workflow and deployment (for the scope of this project)

Reviewer: ```suggestion

```
